jina/logging/logger.py

```diff
@@ -82,14 +82,19 @@ class SysLogHandlerWrapper(logging.handlers.SysLogHandler):
 class JinaLogger:
     supported = {'FileHandler', 'StreamHandler', 'SysLogHandler', 'FluentHandler'}
 
-    def __init__(self, context: str, log_config: Optional[str] = None, **kwargs):
+    def __init__(self, context: str,
+                 log_config: Optional[str] = os.getenv('JINA_LOG_CONFIG',
+                                                       resource_filename('jina', '/'.join(
+                                                           ('resources', 'logging.default.yml')))),
+                 group_id: Optional[str] = os.getenv('JINA_LOGGING_ID', None), **kwargs):
```

Comment:
better make the name consistent, options:
- all to `_LOG_`
- all to `_LOGGER_`
- all to `_LOGGING_`

```suggestion
                 group_id: Optional[str] = os.getenv('JINA_LOG_ID', None), **kwargs):
```

Reply:
also remember to add it into `docs/chapters/envs.rst` this file need a manual maintenance

Reply:
btw, whats the difference between this and 
https://github.com/jina-ai/jina/blob/a2d29300e7fdbda999bf3fe77f3519a5f4de6586/jina/logging/logger.py#L102

? i think it is already there, maybe we just use that?

Reply:
That when we think distributed this name may not be unique

Reply:
Also working with these identifiers can allow the gateway to live in jinad where the identities are known. I think these identities should be provided by them.

Reply:
ok make sense, maybe add `group_id` also to `jina/logging/formatter.py:JsonFormatter` it also outputs structured log